src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GLVectorSizeCache.java

```diff
@@ -0,0 +1,31 @@
+package org.broadinstitute.hellbender.tools.walkers.genotyper;
+
+import java.util.Arrays;
+
+/**
+ * a cache of the PL vector sizes (i.e. the number of possible unphased genotypes) as a function of # of alleles and ploidy
+ *
+ * Created by davidben on 6/9/16.
+ */
+public class GLVectorSizeCache {
```

Comment:
Another consequence of the independent life of the general ploidy code.  I look forward to exercising the delete key.